### PR TITLE
fix(prisma-1/upgrading-the-prisma-layer): Tiny changes to bring up to date

### PR DESCRIPTION
**Pull request by janpio**
Small things I noticed when reading.

### CODE REVIEW

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx [hunk 1]
```diff
@@ -135,18 +135,18 @@ It should print the following output:
   You can now open it in your favorite editor.
 
 Next steps:
-1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started.
+1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
 2. Set the provider of the datasource block in schema.prisma to match your database: postgresql, mysql or sqlite.
-3. Run prisma introspect to turn your database schema into a Prisma data model.
+3. Run prisma db pull to turn your database schema into a Prisma data model.
```
Comment (nikolasburk) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx]:
```suggestion
3. Run `prisma db pull` to turn your database schema into a Prisma data model.
```

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx [hunk 2]
```diff
@@ -135,18 +135,18 @@ It should print the following output:
   You can now open it in your favorite editor.
 
 Next steps:
-1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started.
+1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
```
Comment (nikolasburk) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx]:
```suggestion
1. Set the `DATABASE_URL` in the `.env` file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
```

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx [hunk 3]
```diff
@@ -135,18 +135,18 @@ It should print the following output:
...
 2. Set the provider of the datasource block in schema.prisma to match your database: postgresql, mysql or sqlite.
```
Comment (nikolasburk) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx]:
```suggestion
2. Set the `provider` of your `datasource` block in `schema.prisma` to match your database: `postgresql`, `mysql` or `sqlite`.
```

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx [hunk 4]
```diff
@@ -135,18 +135,18 @@ It should print the following output:
   You can now open it in your favorite editor.
 
 Next steps:
-1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started.
-2. Set the provider of the datasource block in schema.prisma to match your database: postgresql, mysql or sqlite.
-3. Run prisma introspect to turn your database schema into a Prisma data model.
+1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
```
Comment (nikolasburk) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx]:
```suggestion
1. Set the `DATABASE_URL` in the `.env` file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
```

File: content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx [hunk 5]
```diff
@@ -135,18 +135,18 @@ It should print the following output:
...
+2. Set the `provider` of your `datasource` block in `schema.prisma` to match your database: `postgresql`, `mysql` or `sqlite`.
+3. Run `prisma db pull` to turn your database schema into a Prisma data model.
 4. Run prisma generate to install Prisma Client. You can then start querying your database.
```
Comment (nikolasburk) [inline content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx]:
```suggestion
4. Run `prisma generate` to install Prisma Client. You can then start querying your database.
```

**Review by nikolasburk — APPROVED** [review]
Looks great, thanks Jan! 🙏

**Comment by janpio** [comment]
Thanks for the suggestions and fixes, even better 👍

**Pull request merged**